File: src/main/java/rosatech/api/RosaLog.java
package rosatech.api;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
public class RosaLog {
    public static Logger logger = LogManager.getLogger("RosaTech");
    public static Logger modLogger = LogManager.getLogger("RosaTech Mod Integration");

    private RosaLog() {
    }
}

File: src/main/kotlin/rosatech/api/RosaLog.kt
package rosatech.api

import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

object RosaLog {
    var logger: Logger = LogManager.getLogger("RosaTech")
    var modLogger: Logger = LogManager.getLogger("RosaTech Mod Integration")
}
